src/borg/archiver.py

```diff
@@ -4413,12 +4413,12 @@ def get_args(self, argv, cmd):
                     'restrict_to_repositories',
                     'append_only',
                     'storage_quota',
+                    'umask',
                 }
                 whitelist = {
                     'debug_topics',
                     'lock_wait',
                     'log_level',
-                    'umask',
```

Comment:
in case an old client is sending `--umask=XXX`, the server (running this code) would just ignore it and use either the default umask or whatever is given via a ssh forced command.